app/src/main/java/com/lastaoutdoor/lasta/ui/screen/discover/DiscoverScreen.kt

```diff
@@ -356,7 +388,12 @@ fun ActivitiesDisplay(
                       modifier =
                           Modifier.shadow(4.dp, RoundedCornerShape(30))
                               .background(
-                                  MaterialTheme.colorScheme.primary, RoundedCornerShape(10.dp))
+                                  when (a.difficulty) {
```

Comment:
You could add this helper function directly in the `Difficulty` class, in case we need this conversion elsewhere

Reply:
It's a great ideia, I accepted this change request! Now, this is an inner function of the enum Difficulty